internal/services/engines/java/rules.go

```diff
@@ -478,28 +485,31 @@ func NewSQLInjectionWithSqlUtil() text.TextRule {
 		},
 		Type: text.AndMatch,
 		Expressions: []*regexp.Regexp{
-			regexp.MustCompile(`SqlUtil\.execQuery\(`),
+			regexp.MustCompile(`SqlUtil\.execQuery\(.*\+`),
 		},
 	}
 }
 
-func NewNoUseFridaServer() text.TextRule {
-	return text.TextRule{
-		Metadata: engine.Metadata{
-			ID:          "HS-JAVA-27",
-			Name:        "No Use Frida Server",
-			Description: "This App detects frida server.",
-			Severity:    severities.High.ToString(),
-			Confidence:  confidence.High.ToString(),
-		},
-		Type: text.AndMatch,
-		Expressions: []*regexp.Regexp{
-			regexp.MustCompile(`fridaserver`),
-			regexp.MustCompile(`27047|LIBFRIDA`),
-		},
-	}
-}
+// Couldn't find any example or content talking about this vulnerability.
+//
+//func NewNoUseFridaServer() text.TextRule {
```

Comment:
i found this [article](https://notsosecure.com/pentesting-android-apps-using-frida/) and apparently it's really easy to decompile the app and brute force the PIN.

Reply:
Nice! Did you also find something similar to our regex?
Because frida seems to be a pentest tool and I couldn't find anything similar to what our rule is looking for.

```
  regexp.MustCompile(`fridaserver`),
  regexp.MustCompile(`27047|LIBFRIDA`),
```

I decided to comment to avoid false positives, but what do you think?

Reply:
we could change the description to make more explicit why you shouldn't use frida server, maybe fill with some links about this cases